project1.css

```diff
@@ -38,9 +41,44 @@
 
 .intro-page {
 	font-family: Arial;
-	padding-left: 40px;
-	color: white;
+	padding-left: 75px;
+	padding-top: 20px;
+	color: black;
 	background-color: #fc7b03;
 	background-image: linear-gradient(to bottom right, #fc7b03, #000000);
+	width: 100%;
+	height: 100%;
 	}
-}
\ No newline at end of file
+	
+button {
+	background: transparent;
+	border: none;
+}
+
+.about-page {
+	background-color: #000000;
+	width: 100%;
+	height: 100%;
+	font-family: Arial;
+	padding-left: 75px;
+	padding-top: 20px;
+	color: #ffffff;
+}
+
+.about-page p {
+	font-size: 1.5rem;
+	text-align: left;
+	
+}
+
+.contact-page {
+	background-color: #fc7b03;
+	width: 100%;
+	height: 100%;
```

Comment:
I would recommend using 100vh (view height) instead of 100% - view height is predictable (100vh is always the height of the window) whereas 100% can sometimes be smaller or even bigger because it's based on the size of the parent element.